sdk/sdkservices/events.go

```diff
@@ -14,8 +14,14 @@ type ListEventsFilter struct {
 	Limit             int
 	CreatedAfter      *time.Time
 	MinSequenceNumber uint64
+	Order             string
 }
 
+const (
+	ListOrderAscending  = "ASC"
+	ListOrderDescending = "DESC"
+)
```

Comment:
```suggestion
	Order             ListOrder
}

type ListOrder string

const (
	ListOrderAscending  ListOrder = "ASC"
	ListOrderDescending ListOrder = "DESC"
)
```

this way it's clear what the expected values are from the type

Reply:
changed. nice one. I like.